server/sql.go

```diff
@@ -143,6 +143,18 @@ func (s *Server) registerSQLRoutes(g *echo.Group) {
 			}
 		}
 
+		// After syncing schema, we should remove schema drift anomalies.
```

Comment:
The anomaly is produced by the current schema and the schema in the migration history. Syncing database schema doesn’t update any of these two.

Reply:
Then let's don't remove schema drift anomalies after syncing db.